docs/docs/advanced-use/real-time-flags.md

```diff
@@ -63,8 +63,8 @@ but are happy to take pull requests!
 
 ## Things you should know
 
-* Per-identity overrides do not trigger an update
-* Identity Trait updates do not trigger an update
+- Per-identity overrides do not trigger an update
+- Identity Trait updates do not trigger an update
```

Comment:
I'm not sure how this was merged - it's not related to my PR but the linting is broken.

Reply:
I don't see any linting issues?